galaxy_ng/app/settings.py

```diff
@@ -112,3 +114,80 @@
 # Disable django guardian anonymous user
 # https://django-guardian.readthedocs.io/en/stable/configuration.html#anonymous-user-name
 ANONYMOUS_USER_NAME = None
+
+# temporary until env is set in https://github.com/pulp/pulp_installer/pull/719
+os.environ['GALAXY_ENABLE_API_ACCESS_LOG'] = True
+
+if os.environ['GALAXY_ENABLE_API_ACCESS_LOG']:
+    INSTALLED_APPS = ['automated_logging', 'dynaconf_merge']
```

Comment:
same here, `INSTALLED_APPS` already exists on line `17` and here you are redefining that variable.

```suggestion
    INSTALLED_APPS.append('automated_logging')
```